.github/workflows/linux.yml

```diff
@@ -0,0 +1,196 @@
+name: Linux
+
+on:
+  push:
+    branches:
+    - master
+    - staging
+    - trying
+    - release/[0-9]+.[0-9]+
+    tags:
+    - v[0-9]+.[0-9]+.[0-9]+
+  pull_request:
+
+jobs:
+  Lint:
+    runs-on: ubuntu-latest
+
+    if: ${{ github.event_name == 'pull_request' || contains('refs/heads/staging refs/heads/trying', github.ref) }}
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+
+    - name: Run clang-format through the diff
+      uses: ./.github/actions/lint
+
+  Build-Test:
```

Comment:
This was kind of confusing to me, ie, it seems it's building something called "Test".  I don't think spaces are allowed, so maybe something called `BuildAndTest` or `Build-and-Test` or something.  I think having an "and" in there makes it more clear that it's building *and* testing.

Reply:
Done.